debug_module.tex

```diff
@@ -325,9 +328,9 @@ \section{Abstract Commands} \label{abstractcommands}
 \FdmDmcontrolResumereq, \FdmDmcontrolAckhavereset, \FdmDmcontrolSetresethaltreq, or \FdmDmcontrolClrresethaltreq.
 
 If an abstract command does not complete in the expected time and appears to be
-hung, the following procedure can be attempted to abort the command: First the
-debugger resets the hart (using \FdmDmcontrolHartreset or \FdmDmcontrolNdmreset), and then it resets
-the Debug Module (using \FdmDmcontrolDmactive).
+hung, the debugger can try to reset the hart (using \FdmDmcontrolHartreset or
+\FdmDmcontrolNdmreset). If that doesn't clear \FdmAbstractcsBusy, then it
+can try resetting the Debug Module (using \FdmDmcontrolDmactive).
```

Comment:
Wouldn't resetting the debug module always recover from this hang condition?  Not sure that "can try resetting" is the best terminology ... maybe "...then it must reset the Debug Module using..."

Reply:
I could imagine some bug/condition in the hart itself causing an abstract command to hang, and resetting the DM does not reset the hart so wouldn't necessarily resolve that.